index.html

```diff
@@ -172,51 +172,31 @@ <h2>Languages</h2>
       </section>
     </main>
 
-
-    <footer class="footer" id="contact">
-        <div class="contact-content">
-          <h2 class="contact-header">Contant Us</h2>
-          <p class="contact-paragraph">
-            If you have an application you are interested in developing, a feature
-            that you need built or a project that needs coding. I’d love to help
-            with it
-          </p>
-          <div class="footer-items">
-            <form action="https://formspree.io/f/xjvlllyy" method="POST">
-              <label class="label-visibility" for="user-name">Name</label>
-              <input
-                name="name"
-                id="user-name"
-                type="text"
-                placeholder="Name"
-                maxlength="30"
-                required
-              />
-            <label class="label-visibility" for="user-email">Email</label>
-              <input
-                name="email"
-                id="user-email"
-                type="email"
-                placeholder="name@gmail.com"
-                required
-              />
-            
-            <label class="label-visibility" for="msg">Message</label>
-              <textarea
-                id="msg"
-                name="message"
-                maxlength="500"
-                required
-                placeholder="Write your message here"
-              ></textarea>
-              <button type="submit" class="footer-button" >Get in Touch</button>
-            </form>
-            <img class="footer-img" src="image/Shape.png" alt="" />
-        </div>
-        </div>
-      </footer>
-      <section class="worksPopup" id="portfolioPopup">
-      </section>
+<section class="contact" id="contact">
+  <div class="inner-div">
+    <h2>Contact me</h2>
+    <p id="inner-div">If you have an application you are interested in developing, a feature that you need
+      built or a project that needs coding. I’d love to help with it </p>
+  </div>
+  <form action="https://formspree.io/f/xbjwrzvy" id="form" class="form" method="post">
+    <ul class="inner-list">
+      <li><input type="text" class="form-control" placeholder="Full name" name="name" id="name" size="40"
+          maxlength="30">
+      </li>
+      <li> </li>
+      <li><input type="email" class="form-control" name="email" id="email" placeholder="Email address">
+      </li>
+      <li> </li>
```

Comment:
- [ ] The error message should show up close to the `submit` button.
### **Your current case**
![emailE](https://user-images.githubusercontent.com/55596970/174241072-3f1af3aa-cef2-4175-b1a2-6e8abe4ee635.png)

Reply:
- Fixed the background image issue
- The location of errors are okay now(Every error message is properly located)
<img width="1425" alt="Screen Shot 2022-06-17 at 6 44 33 PM" src="https://user-images.githubusercontent.com/53530780/174316880-20da0219-7ac9-4a1c-96fa-39cf4cba544b.png">

Reply:
@shahira-sadat , as pointed out by the previous reviewer, the location of the error message should be close to the submit button. However, it shows on top of the text area. Here is the reference.
![image](https://user-images.githubusercontent.com/33268927/174319689-63c63079-2333-4c1e-b888-0a54b2e8ee77.png)

Reply:
Hi @shahira-sadat 

- [ ] Please the error message for the email should show up close to the submit button.